### DISCUSSION TITLE
Shuffle during CV

### DISCUSSION THREAD

**Discussion by Sukantabasu** [microsoft/FLAML#490] [Q&A]
I am using eval_method = cv with n_splits = 5. By default the "uniform split" option is selected. I am not sure if FLAML is shuffling data prior to split. I am expecting it not to.

**Comment by sonichi** [comment]
It shuffles the data prior to split. You can use a custom splitter as "split_type" to prevent the shuffling if you prefer the data not to be shuffled: https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#data-split-method

**Reply by Sukantabasu** [reply]
Thanks Chi for your prompt response. I will use a custom splitter. Maybe in a future version, shuffle can be used as a boolean option.

**Reply by sonichi** [reply]
Good suggestion. It'll be great if you could create a PR or an issue together with the custom splitter that works for you.
